tensorboard/webapp/core/views/layout_container.ts

```diff
@@ -55,9 +59,15 @@ import {getSideBarWidthInPercent} from '../store/core_selectors';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class LayoutContainer implements OnDestroy {
-  readonly width$: Observable<number> = this.store.select(
-    getSideBarWidthInPercent
-  );
+  readonly runsTableFullScreen$ = this.store.select(getRunsTableFullScreen);
+  readonly width$: Observable<number> = this.store
+    .select(getSideBarWidthInPercent)
+    .pipe(
+      combineLatestWith(this.runsTableFullScreen$),
```

Comment:
Clearly this works. However, I thought combineLatestWith would cause this pipe to not trigger an update when runsTableFullScreen changes.